deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/security/TokenAuthFilter.java

```diff
@@ -52,9 +52,9 @@ public TokenAuthFilter(String userDataUrl, String groupDataUrl, String userNameK
     public void filter(ContainerRequestContext context) throws IOException {
         if(!context.getMethod().equals("OPTIONS")) {
             SecurityContext securityContext;
-
             try {
                 securityContext = authenticate(context);
+                LOG.info("request method: " + context.getMethod() + "; request uri: " + context.getUriInfo().getRequestUri() + "; user name: " + securityContext.getUserPrincipal().getName());
```

Comment:
Could you format the string in json format so it's easier to query when needed? 
i.e.
```
LOG.info("\"method\":{}, \"user\": {}", method, user);
```